linkedlist/app/src/main/java/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,47 @@
+package linkedlist;
+
+public class LinkedList {
+    Node head;
+
+
+    public void insert(Object value){
+           Node newNode = new Node(value);
+           newNode.next = this.head;
+           this.head = newNode;
+
+    }
+    public boolean includes(Object value) {
+        Node currentNode = this.head;
+
+        boolean result = false;
```

Comment:
Not sure why you declare result here? You have a method which expects a bool, you can just return T/F if you find the value.